src/hydra/hydra-plugin.js

```diff
@@ -185,19 +181,26 @@ export class HydraPlugin {
     }
   }
 
-  async getRelation({state}, itemOrIri, options) {
+  async getRelation({state}, itemOrIri, options = {}) {
     if (null === itemOrIri) {
       return null;
     }
 
-    if ('object' === typeof itemOrIri) {
-      return itemOrIri;
-    }
-
     if ('function' === typeof itemOrIri) {
       return computedAsPromise(() => this.getRelation({state}, itemOrIri(), options));
     }
 
+    if (true === (options.useExisting ?? true)) {
```

Comment:
Maybe add tests for `useExisting` and `force` options?

Reply:
Done.